static/js/search.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from "react";
+import axios from "axios";
+import { updateUrlPlace } from "./util";
+
+let ac: google.maps.places.Autocomplete;
+
+interface PlacePropType {
```

Comment:
ChpPropType. The name should follow "ComponentName", "ComponentNamePropType", "ComponentNameStateType"

Reply:
done.